front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -197,19 +198,25 @@ public boolean supportsVersioning() {
     }
   }
 
+  @Override
+  public <T extends Timestamped> Collection<T> loadObjectsWithPrefix(ObjectType objectType, String prefix, int maxResults) {
+    throw new UnsupportedOperationException("loadObjectsWithPrefix is not yet supported!");
```

Comment:
This will need a GCS implementation @ewiseblatt ... only used by `EntityTags` so not breaking any existing functionality.